cli/src/config/misc.toml

```diff
@@ -6,6 +6,9 @@ b = ["bookmark"]
 co = ["checkout"]
 unamend = ["unsquash"]
 
+[diff]
+context = 3
```

Comment:
Since we have `[diff.<format>]` namespace, I think it's better to add `diff.color-words.context` and `diff.git.context` separately.